mkstage4.sh

```diff
@@ -65,7 +66,12 @@ while getopts ':t:e:skqcblph' flag; do
       EXCLUDE_LOST=1
       ;;
     e)
-      USER_EXCL+=" --exclude=${OPTARG}"
+      if [ -e ${OPTARG} ]
```

Comment:
`OPTARG` is likely to contain `*` since --exclude accepts such values in order to glob.

Also the arg to --exclude is relative to `TARGET`. This will be an issue when `TARGET` isn't `/` nor `$(pwd)`.